subprojects/studio-plugin/src/test/groovy/org/gradle/profiler/studio/plugin/StudioPluginIntegrationTest.groovy

```diff
@@ -38,14 +40,14 @@ class StudioPluginIntegrationTest extends StudioPluginTestCase {
         given:
         ServerConnection connection = server.waitForIncoming(Duration.ofSeconds(10))
         // Fail Gradle build by removing settings file
-        settingsFile.delete()
+        settingsFile << "garbage"
 
         when:
         connection.send(new StudioRequest(SYNC))
 
         then:
         StudioSyncRequestCompleted requestCompleted = connection.receiveSyncRequestCompleted(Duration.ofSeconds(90))
-        requestCompleted.result == FAILED
+        assert requestCompleted.result == FAILED
```

Comment:
Why do you need an `assert` here?

Reply:
It seems Spock is not wired correctly somehow, so it doesn't enhance statement with asserts.